app/screens/reschedule_draft/reschedule_draft.test.tsx

```diff
@@ -189,4 +194,48 @@ describe('RescheduledDraft', () => {
         // Verify dismissModal was called
         expect(dismissModal).toHaveBeenCalledWith({componentId: baseProps.componentId});
     });
+
+    it('should show snackbar when no time is selected', async () => {
```

Comment:
This is for when the time is not selected. Should we add any other tests regarding the other places where the snackbar should show (and not show)?